pkg/operator/resources/cluster/cdiconfig.go

```diff
@@ -36,497 +36,263 @@ func createCDIConfigCRD() *extv1.CustomResourceDefinition {
 					Name:    "v1alpha1",
 					Served:  true,
 					Storage: false,
-					Schema: &extv1.CustomResourceValidation{
-						OpenAPIV3Schema: &extv1.JSONSchemaProps{
-							Description: "CDIConfig provides a user configuration for CDI",
+					Schema:  createCDIConfigCRDSchema(),
```

Comment:
Right here we assume that alphav1 and betav1 always remain the same, which is currently the case, but once we convince KubeVirt to use betav1, we won't update alphav1 anymore. So we would need to diverge the two at that point. Maybe start that by having a createAlphaV1ConfigCRDSchema and createBetaV1ConfigCRDSchema, and for now have alpha call beta, and put in a note that they might start diverging.

Reply:
@awels done.